MiniProject1.py

```diff
@@ -0,0 +1,96 @@
+# Kimber's file
+from pattern.en import *
+rhymeWords=dict() #dictionary of words that rhyme
+generalWords=dict() #markov chain dictionary
+#fo = open("ShakespeareSonnets.txt", "r")
+with open('ShakespeareSonnets.txt', 'r') as f:
+	read_data = f.read()
+f.closed
+sonnetz= read_data.split('\n\n')
+sonnetLine=[0, 1, 2, 3,4, 5,6 ,7, 8, 9, 10, 11, 12, 13]
+import random
+
+for k in range(0,len(sonnetz)-1):
+	lines=sonnetz[k].split('\n') #splits into lines
+
+	def DictRhymes():
```

Comment:
If there's a method only defined inside of a for loop, that's awful coding hygiene and I'm deeply ashamed and going to fix that, cuz my mama didn't raise me to write code like that.